packages/components/input/src/use-input.ts

```diff
@@ -370,7 +370,7 @@ export function useInput<T extends HTMLInputElement | HTMLTextAreaElement = HTML
         "data-has-start-content": dataAttr(hasStartContent),
         "data-has-end-content": dataAttr(!!endContent),
         className: slots.input({
-          class: clsx(classNames?.input, isFilled ? "is-filled" : ""),
+          class: clsx(classNames?.input, isFilled ? "is-filled" : "", isMultiline ? "pr-0" : ""),
```

Comment:
seems not required

Reply:
pr-0 is because pr-6 is added to the input or textarea element if isClearable is true, which is the unexpected padding problem you mentioned above